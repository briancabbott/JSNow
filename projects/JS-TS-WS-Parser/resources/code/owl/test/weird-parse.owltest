---
e=b b b b b b b '0' b b
b='' : loooooooooooong '0' : b
