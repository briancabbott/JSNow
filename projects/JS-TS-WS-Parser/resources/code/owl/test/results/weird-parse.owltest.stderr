error: this grammar is ambiguous

. 0 0 

  can be parsed in two different ways: as

.                                                                        0 0  ..
  b:loooooooooooong b:loooooooooooong b:loooooooooooong b:loooooooooooong       
  e-------------------------------------------------------------------------    
 .. 

  or as

.                                                                        0 0  ..
  b:loooooooooooong b:loooooooooooong b:loooooooooooong b:loooooooooooong       
  e-------------------------------------------------------------------------    
 .. 

